lib/models/blueprint.js

```diff
@@ -850,7 +850,9 @@ Blueprint.prototype._locals = function(options) {
   var sanitizedModuleName = moduleName.replace(/\//g, '-');
 
   return new Promise(function(resolve) {
-    resolve(this.locals(options));
+    return resolve(this._processPrompts(options));
+  }.bind(this)).then(function (prompts) {
```

Comment:
TBH I'd actually prefer a different syntax here but @stefanpenner will probably be able to tell if it has any disadvantages:

```js
return Promise.resolve().then(function() {
  return this._processPrompts(options);
}.bind(this)).then(function(options) {
  return this.locals(options);
}.bind(this)).then(/* ... */);
```